<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_8</class>
 <widget class="QDialog" name="fct_comp_8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les réservations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_ajout_fct_comp_8">
     <property name="title">
      <string>Ajouter une réservation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="ajout_VLayout">
        <item>
         <widget class="QLabel" name="label_dos_fct_comp_8">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Dossier courant : n°?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="ajout_HLayout">
          <item>
           <layout class="QVBoxLayout" name="layout1_spec_fct_comp_8">
            <item>
             <widget class="QLabel" name="label_spec_fct_comp_8">
              <property name="text">
               <string>Spectacle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_spec_fct_comp_8">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout2_date_fct_comp_8">
            <item>
             <widget class="QLabel" name="label_date_fct_comp_8">
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_date_fct_comp_8">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout3_rang_fct_comp_8">
            <item>
             <widget class="QLabel" name="label_rang_fct_comp_8">
              <property name="text">
               <string>Rang</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_rang_fct_comp_8">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout4_plac_fct_comp_8">
            <item>
             <widget class="QLabel" name="label_plac_fct_comp_8">
              <property name="text">
               <string>Place</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_plac_fct_comp_8">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout5_redu_fct_comp_8">
            <item>
             <widget class="QLabel" name="label_redu_fct_comp_8">
              <property name="text">
               <string>Réduction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_redu_fct_comp_8">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout6_action_fct_comp_8">
            <item>
             <widget class="QPushButton" name="pushButton_newdos_fct_comp_8">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Nouveau dossier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ajout_fct_comp_8">
              <property name="text">
               <string>Ajouter</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_tri_action">
     <item>
      <widget class="QGroupBox" name="groupBox_tri_fct_comp_8">
       <property name="title">
        <string>Options de tri</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_tri_dos_fct_comp_8">
          <property name="text">
           <string>Par numéro de dossier</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_tri_spec_fct_comp_8">
          <property name="text">
           <string>Par représentation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_update_fct_comp_8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete_fct_comp_8">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_8">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>noDos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomSpec</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateRep</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noPlace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noRang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>libelleCat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateEmTick</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>prixTicket</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error_fct_comp_8">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_delete_fct_comp_8">
     <property name="title">
      <string>Confirmation de la suppression</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_delmsg_fct_comp_8">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Êtes-vous sûr de vouloir supprimer la réservation suivante ?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_delrow_fct_comp_8">
        <property name="text">
         <string>---</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_delete_conf">
        <item>
         <widget class="QPushButton" name="pushButton_ndel_fct_comp_8">
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ydel_fct_comp_8">
          <property name="text">
           <string>Confirmer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_tri_dos_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_tri_spec_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_spec_fct_comp_8</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshDateList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_date_fct_comp_8</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshRowList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_rang_fct_comp_8</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_8</receiver>
   <slot>refreshPlacList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ajout_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>addReservation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_newdos_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>addFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>585</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_delete_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>deleteReservation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ndel_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>deleteNo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>719</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ydel_fct_comp_8</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_8</receiver>
   <slot>deleteYes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>696</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>refreshDateList()</slot>
  <slot>refreshRowList()</slot>
  <slot>refreshPlacList()</slot>
  <slot>addReservation()</slot>
  <slot>refreshCurrentFolder()</slot>
  <slot>addFolder()</slot>
  <slot>deleteReservation()</slot>
  <slot>deleteYes()</slot>
  <slot>deleteNo()</slot>
 </slots>
</ui>
